### PR TITLE
PLNSRVCE-1179: CI test the downgrade path

### PR DESCRIPTION
**Pull request by xinredhat**
A new CI test the downgrade path

### CODE REVIEW

File: .tekton/pipeline-service-downgrade-test.yaml [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+            value: $(params.target_branch)
+        taskSpec:
+          workspaces:
+            - name: kubeconfig-dir
+            - name: source
+          params:
+            - name: repo_url
+            - name: revision
+            - name: target_branch
+          kind: ClusterTask
```
Comment (ramessesii2) [inline .tekton/pipeline-service-downgrade-test.yaml:193]: is there a reason for this task to be a `ClusterTask`?

**Review by ramessesii2 — APPROVED** [review]
lgtm

File: .tekton/pipeline-service-downgrade-test.yaml (hunk 1 above)
Comment (xinredhat) [reply .tekton/pipeline-service-downgrade-test.yaml]: I removed them and lets see the CI results

**Pull request merged**